torch/utils/_triton.py

```diff
@@ -2,6 +2,19 @@
 import hashlib
 
 
+@functools.lru_cache(None)
+def device_supports_triton_float8() -> bool:
+    import torch
+
+    n = torch.cuda.device_count()
+    if n < 1:
+        return False
+    capabilities = {torch.cuda.get_device_capability(i) for i in range(n)}
+    min_capability = sorted(capabilities)[0]
+    # triton requires CUDA arch for native float8 support >= 89
+    return min_capability[0] >= 8 or (min_capability[0] == 8 and min_capability[1] >= 9)
```

Comment:
wasn't sure if there's a more canonical way to do this (@eellison if you happen to know a better way?)

Reply:
`min_capability >= (8, 9)`?